tests/testMatchFits.py

```diff
@@ -44,20 +44,20 @@ def setUp(self):
         self.cat1 = afwTable.SimpleCatalog(self.schema)
         self.cat2 = afwTable.SimpleCatalog(self.schema)
         for i in range(self.size):
-            record1 = self.cat1.table.makeRecord()
-            record2 = self.cat2.table.makeRecord()
+            record1 = self.cat1.makeRecord()
+            record2 = self.cat2.makeRecord()
```

Comment:
Omost forwarded methods, I'd agree with @r-owen, but this particular one is a reminder that maybe we shouldn't be forwarding *all* of Table's methods to Catalog (though I'm not suggesting we try to fix that now).  `catalog.makeRecord` is a bit confusing, because it  creates a record without adding it to the catalog, so I think this was probably better as it was.